src/Ease.elm

```diff
@@ -73,28 +73,93 @@ linear =
 
 {-| A cubic bezier function using 4 parameters: x and y position of first control point, and x and y position of second control point.
 
-See [here](http://greweb.me/glsl-transition/example/ "glsl-transitions") for examples or [here](http://cubic-bezier.com/ "tester") to test.
+See [here](http://cubic-bezier.com/ "tester") to test.
```

Comment:
I removed this link since it does not work anymore. Are there other resources we could link to?